include/boost/spirit/home/x3/support/utility/error_reporting.hpp

```diff
@@ -158,7 +158,15 @@ namespace boost { namespace spirit { namespace x3
         for (Iterator i = first; i != pos; ++i)
             if (*i == '\r' || *i == '\n')
                 latest = i;
```

Comment:
I am pretty sure that the only needed fix for the function is `latest = i;` -> `latest = ++i;`.